### PR TITLE
Created dialog for course team contact payment teaser

### LINKED ISSUE #2469: Show teaser dialog when user has not paid and attempts to contact course team
In an effort to move things along and keep the PR more manageable, #2284 was done without the modal that we want to show users who haven't paid for a course and want to contact the course team ([link to the dialog design](https://github.com/mitodl/micromasters/issues/2284#issuecomment-274866731)). The work for this issue will be to implement that dialog.

### PR DESCRIPTION
**Pull request by gsidebo**
#### What are the relevant tickets?

Closes #2469 

#### What's this PR do?

Enables the 'Contact course team' link for unpaid users and shows them a message that they need to pay.

#### Where should the reviewer start?

DashboardPage.js

#### How should this be manually tested?

Set `MAILGUN_RECIPIENT_OVERRIDE` to your email, then check the behavior and appearance of the 'Contact course team' link on the dashboard with the following scenarios:

1. The user is enrolled & paid, and the course has a contact email
1. The user is enrolled but unpaid, and the course has a contact email
1. The course has no contact email

#### Screenshots (if appropriate)

ss 2017-02-02 at 12 44 46

### CODE REVIEW

File: static/js/containers/DashboardPage.js [hunk 1]
```diff
@@ -438,6 +450,25 @@ class DashboardPage extends React.Component {
     dispatch(setRecentlyAttachedCoupon(coupon));
   };
 
+  navigateToProfile = () => {
```
Comment (gsidebo) [inline static/js/containers/DashboardPage.js]: i moved this function and the one below to keep the `render*` functions together

**Comment by roberthouse54** [comment]
The image is a bit too big... Can you make it 65% the size it is now?

**Comment by noisecapella** [comment]
Can you rebase please?

**Comment by noisecapella** [comment]
It would be nice to use an SVG version of the contact course team icon, not sure if that's available though

File: static/js/actions/ui.js [hunk 2]
```diff
@@ -78,6 +78,9 @@ export const setSearchFilterVisibility = createAction(SET_SEARCH_FILTER_VISIBILI
 export const SET_EMAIL_DIALOG_VISIBILITY = 'SET_EMAIL_DIALOG_VISIBILITY';
 export const setEmailDialogVisibility = createAction(SET_EMAIL_DIALOG_VISIBILITY);
 
+export const SET_PAYMENT_TEASER_DIALOG_VISIBILITY = 'SET_PAYMENT_TEASER_DIALOG_VISIBILITY';
```
Comment (noisecapella) [inline static/js/actions/ui.js]: Can you update `ui_test.js` to include this action?

File: static/js/components/dashboard/CourseDescription_test.js [hunk 3]
```diff
@@ -30,12 +30,18 @@ describe('CourseDescription', () => {
...
+    courseRun,
+    courseTitle,
+    canContactCourseTeam = false,
+    hasContactEmail = true
+  ) => (
     shallow(
       <CourseDescription
         courseRun={courseRun}
         courseTitle={courseTitle}
         canContactCourseTeam={canContactCourseTeam}
```
Comment (noisecapella) [inline static/js/components/dashboard/CourseDescription_test.js]: Can you remove this prop and the corresponding argument in this function?

**Comment by gsidebo** [comment]
@noisecapella should be all set

**Pull request merged**